packages/mobile/src/utils/calendarTools.ts

```diff
@@ -108,3 +110,10 @@ export const getCalendarMap = (
   });
   return calendarMap;
 };
+
+export const caculateDateNum = (year: number, month: number) => {
+  const DATE_NUM = [ 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 ];
+  if (month === 1 && ((year % 4 === 0 && year % 100 !== 0) || year % 400 === 0))
+    return 29;
+  return DATE_NUM[month];
+};
```

Comment:
혹시 `dayjs` 라이브러리 적용하시면 이렇게 줄일 수 있을 것 같습니다~

```suggestion
export const caculateDateNum = (year: number, month: number) => dayjs().year(year).month(month).date(0).date();
```